api/v1alpha1/release_conditions.go

```diff
@@ -6,6 +6,10 @@ const (
 	// deployedConditionType is the type used to track the status of a Release deployment
 	deployedConditionType conditions.ConditionType = "Deployed"
 
+	// matchedConditionType is the type used to track the status of the ReleasePlan being matched to a
```

Comment:
This should be in a releaseplan_conditions.go or releaseplanadmission_conditions.go file.